cql/DashboardLibrary.cql

```diff
@@ -1247,8 +1296,17 @@ define Errors:
 // SUMMARY
 
 define Pregnant:
-  Exists(PregnancyDiagnoses) or
-  Exists(PregnancyObservations)
+  // IF any Epsidose of care with status = active
+  //    Pregnant = true
+  // If any Episode of care with status = finished
+  //    Pregnant = false
+  // Use existing logic with conditions and observations
+  if (HasActivePregnancyEpisodesOfCare) then
+    true
+  else if (HasFinishedPregnancyEpisodesOfCare and not HasActivePregnancyEpisodesOfCare) then
```

Comment:
Don't need "and not HasActivePregnancyEpisodesOfCare" because it has been taken care of by the first if condition

Reply:
Good catch. Removed the redundant statement in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/119/commits/5cf166def57145008b5b59651cedcb4e8db53a75.